index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="container">
+       <a class="name">John Doe</a>
+        <div class="hamburger"> 
+            <img src="icons/Union.png" alt="Hamburger menu icon">
+        </div>
+    </header>
+<section class="headline">
+    <h1 class="title">Hey There.
+        I'm Alex</h1>
+    <h3 class="subtitle">I'm a Software Developer</h3>
+    <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don't hestiate to contact me.</p>
+    </section>
+<ul class="icons">
+    <li><a href="https://github.com/Daf98"></a><img src="icons/Github.png" alt="Github icon"></li>
+    <li><a href="https://linkedin.com/in/dafne-azzolina/"></a><img src="icons/Linkedin.png" alt="LinkedIn icon"></li>
+    <li><a href="https://angel.co/"></a><img src="icons/Angelist.png" alt="AngelList icon"></li>
+    <li><a href="https://twitter.com/dafne_azzolina"></a><img src="icons/Twitter.png" alt="Twitter icon"></li>
+    <li><a href="https://medium.com/"></a><img src="icons/Medium.png" alt="Medium icon"></li>
```

Comment:
In this case, the anchor and the image are siblings, when the image, the icon should be the child of the anchor.

- [ ] Kindly, use the example provided by the previous reviewer and correct the above-mentioned issue;